src/main/java/com/capgemini/fs/coindashboard/controller/exceptionHandler/InvalidPathAdvice.java

```diff
@@ -0,0 +1,18 @@
+package com.capgemini.fs.coindashboard.controller.exceptionHandler;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class InvalidPathAdvice {
```

Comment:
Maybe would be better to create base exception that will be used for all other controller exceptions like: InvalidPathException
In that approach we do not need to create ControllerAdvice for all other controller exceptions, like it's done for here: InvalidPathAdvice. We can create one ControllerAdvice that will handle all controller exceptions that based on our base exception